analyzer/src/main/kotlin/curation/ClearlyDefinedPackageCurationProvider.kt

```diff
@@ -101,6 +103,12 @@ class ClearlyDefinedPackageCurationProvider(server: Server = Server.PRODUCTION)
                 }
             }
 
+            return emptyList()
+        } catch (e: MismatchedInputException) {
+            e.showStackTrace()
+
+            log.warn { "Deserializing the curation for '${pkgId.toCoordinates()}' failed." }
```

Comment:
I think you should make it clear that is a ClearlyDefined curation that failed.
Without context this warning may cause confusion.